orchestrator/src/test/java/com/scottlogic/deg/orchestrator/cucumber/features/operators/grammatical/If.feature

```diff
@@ -1,5 +1,7 @@
 Feature: Values can be specified by using if, then and else constraints
 
+  # this feature is a duplicated in IfDecisionBased.feature. this one tests the reductive walker
```

Comment:
This comment can be removed if the tests are just going to run on the new walker.